elements/lisk-p2p/test/utils/network_setup.ts

```diff
@@ -57,6 +63,15 @@ export const createNetwork = async ({
 	networkDiscoveryWaitTime,
 	customConfig,
 }: TestNetworkConfig = {}) => {
+	const serverSocketPrototype = SCServerSocket.prototype as any;
```

Comment:
I don't like to bring this to the general setup. I'd prefer to keep it just where it's needed. Would it be possible?